lms/envs/common.py

```diff
@@ -2477,6 +2477,9 @@ def _make_locale_paths(settings):  # pylint: disable=missing-docstring
 
     # Management of external user ids
     'openedx.core.djangoapps.external_user_ids',
+
+    # Management of per-user schedules
+    'openedx.core.djangoapps.schedules',
```

Comment:
Should the entry point also be removed from setup.py?  Although that also adds it to Studio.

Reply:
It also registers some signals. Although, if it's being added as a plugin in studio, then we should install it there too.

Reply:
Yeah, I'll de-plugin-ify it.